index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"><link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css">
+    <title>Capstone Project-Sozo Graphics Institute</title>
+</head>
+<body id="body">
+    <section class="header">   
+        <div class="nav-container">
+            <div class="menu_btn" onclick="openMobileMenu()">
+                <img src="images/menu-rounded.png" class="hamburger_menu"  alt="menu button">
+            </div>
+                <div class="top-nav">
+                    <ul>
+                    <li><a href="#" target="_blank"><i class="fa-brands fa-facebook-f"></i></a></li>
+                    <li><a href="#" target="_blank"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">LogOut</a></li>
+                    </ul>
+                </div>
+            
+                <nav class="navigation" id="nav">
```

Comment:
- [ ] [OPTIONAL] Currently the links on the mobile menu are hard to see because they're overlapping with black background on some parts of the page. I think it would be great if you give them a different color. 

<img src="https://user-images.githubusercontent.com/17243325/217492652-6378aa21-bcbc-4456-825f-58c40afd5df1.png" width="300">